usort/api.py

```diff
@@ -61,8 +79,16 @@ def usort_string(data: str, config: Config, path: Optional[Path] = None) -> str:
 
     - a string unrepresentable in utf-8, e.g. "\ud800" is a single high surrogate
     - a string with a valid pep 263 coding line, other than utf-8
+
+    DEPRECATED: use `usort()` directly, and and encode/decode bytes as necessary.
     """
-    return usort_bytes(data=data.encode(), config=config, path=path).output.decode()
+    warn("use usort() instead", DeprecationWarning, stacklevel=2)
```

Comment:
Why 2?

Reply:
With `stacklevel=1`, the deprecation warning shows the line of code that calls `warn()`. With `stacklevel=2`, the warning shows the line of code that calls `usort_string()` or `usort_bytes()`.